examples/laptop_complete_pipeline.yml

```diff
@@ -6,7 +6,12 @@ stages:
     - name: FlexZPipe
       #- name: PZPDFMLZ
     - name: TXPhotozStack
-    - name: TXDiagnosticMaps
+    - name: TXMainMaps
```

Comment:
This is general for all the stages -- now that we are accumulating more and more stages and it's less clear what each stages does from the name. Maybe put a short comment after each of them here?

Reply:
good idea - I've done this in the main YAML file for now.